### DISCUSSION TITLE
Repository Creation

### DISCUSSION THREAD

**Discussion by fosskers** [magit/forge#752] [Ideas]
The official `gh` tool allows for new repository creation on Github via their API. It would be handy if this were possible in Forge too (I tried it this morning and found the feature missing).

For instance you can run:

``` sh
gh repo create --private
```

and it runs you through a few prompt steps setting the description, etc., and then after getting your SSH key creates the repository remotely without ever visiting Github itself.

Thank you!

**Comment by tarsius** [comment]
I plan to move away from forges such as Github. I was planning to move to Codeberg, but am not sure anymore, that is enough. Developments over the last few and especially the last year, forced me to invest in defense, and I am increasingly worried that having a world-writable public todo list puts maintainers under constant stress. I'll keep maintaining Forge; after all most (i.e., currently all) of my packages are still hosted on Github and I also want to keep using it for repositories on Github, which are owned by other people. But new features are becoming increasingly unlikely.

So... creating *new* repositories on Github, isn't exactly something I want to encourage, and thus do not want to make easier.

**Reply by fosskers** [reply]
Understood, thanks.
